blogposts/2021/better-onboarding-advice-from-3-engineering-leaders.md

```diff
@@ -0,0 +1,127 @@
+They may not even be familiar with the general toolset that your team has come to rely on to work efficiently.
+
+Great companies invest a great deal of effort into helping new engineers onboard to their codebases. 
+In this post, we'll take a deep dive with engineering leaders from 3 highly reputed engineering organizations to understand how they structure their onboarding, and how they address specific challenges in order to support their new hires and help them ramp up as quickly as possible.
+
+[Limor Bergman](https://www.linkedin.com/in/limorbergman/) is a former Director of Engineering at Digital Ocean, where she led the organizations responsible for Droplets, Kubernetes, Kernel, Performance, Systems, and Orchestration. 
+She's now the director of the mentorship program at PowerToFly, a non-profit dedicated to helping women grow their careers.
+
+[Ryan Djurovich](https://www.linkedin.com/in/ryandjurovich/) is a DevOps Manager at Xero, Australia's largest accounting software company. 
+He leads the team responsible for developer infrastructure, experience, and tooling.
+
+[Jean du Plessis](https://www.linkedin.com/in/jeandp/) is a Director of Engineering at Sourcegraph, where he leads the Developer Insights organization, which includes the frontend platform, extensibility, code insights, and API docs teams.
+
+The 3 offered their thoughts on the process of onboarding new engineers to the team, revealing some strong parallels as well as some particular areas of emphasis and focus.
+
+## Help new hires see the bigger picture with multi-tiered onboarding
+
+Multi-tiered onboarding provides a layered, holistic view of the organization.
+
+All 3 engineering leaders described a multi-tier onboarding process with steps both common across the company and organization and ones specific to the new hire's immediate team. 
+Some things should be standard across the organization: setting up and familiarizing oneself with shared collaboration tools (email, the code host, chat, CI/CD, the code review) and understanding the "big picture" of what the company does and how the various organizations interact with each other. 
+Department heads can give regular presentations to new hires about what their organization does. 
+
+Individual teams should also think about how to onboard new members to the things that are specific to the team, such as setting up the development environment that members of the team use, meeting and establishing relationships with teammates, and understanding what’s necessary to ship that first bug fix or feature enhancement.
+
+## Share high-level context
+
+High-level context is the hardest to transfer. It’s the hardest to document (without going out of date) and you can't automate it, because this really just reduces to grokking the overall architecture, which isn’t the same as truly understanding how the pieces fit together and knowing what the core assumptions and tension points are. 
```

Comment:
```suggestion
High-level context is the hardest to transfer. It’s difficult to document, because grokking the overall architecture means understanding how the pieces fit together and knowing what the core assumptions and tension points are. Even if you do a reasonable job with the documentation, chances are it will fall out of date.
```